proto/alpine.proto

```diff
@@ -0,0 +1,26 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+syntax = "proto3";
+
+package alpine;
+
+option go_package = "github.com/openconfig/kne/proto/alpine";
+
+// Alpine specific vendor data for KNE
+message AlpineDataplaneConfig {
```

Comment:
I think it would be easier to expand this in the future (if needed for any other custom parameters) if this was just called AlpineConfig, something like:

```
message AlpineConfig {
    repeated Container containers = 1;
}

message Container {
    string name = 1;
    ...
}
```

or some variation of the above as long as the message passed in the textproto is AlpineConfig rather than AlpineDataplaneConfig

Reply:
Thanks for the suggestion, done!